quiz/main.go

```diff
@@ -12,29 +15,53 @@ type Task struct {
 }
 
 func getTasks(filename string) (tasks []Task, err error) {
-	content_bytes, err := ioutil.ReadFile(filename)
-	content := string(content_bytes)
+
+	file, err := os.Open(filename)
 	if err != nil {
 		return nil, err
 	}
+	defer file.Close()
 
-	lines := strings.Split(content, "\n")
+	csv_reader := csv.NewReader(file)
 
-	for _, line := range lines {
-		if line == "" {
-			continue
+	for {
+		record, err := csv_reader.Read()
+		if err == io.EOF {
+			break
+		}
+		if err != nil {
+			return nil, err
 		}
-		var values = strings.Split(line, ",")
-		tasks = append(tasks, Task{values[0], values[1]})
+
+		tasks = append(tasks, Task{record[0], record[1]})
 	}
 
 	return tasks, nil
 }
 
 func main() {
-	var tasks, _ = getTasks("./problems.csv")
+	var filename string
+	var limit int
+
+	flag.StringVar(&filename, "csv", "./problems.csv", "a csv file in the format of 'question,answer' (default \"problems.csv\"")
+	flag.IntVar(&limit, "limit", 30, "the time limit for the quiz in seconds (default 30)")
+
+	flag.Parse()
+
+	var tasks, _ = getTasks(filename)
+	// TODO: add error processing
+
+	var correct = 0
+	var answer string
```

Comment:
It is more common to define variables either with `:=` or with `var` block 
```suggestion
	var (
	    correct  =  0
	    answer string
	)
```